src/Python/Meshes/Decimation.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+import os.path
+
+import vtk
+
+
+def get_program_parameters():
+    import argparse
+    description = 'Decimate polydata.'
+    epilogue = '''
+    This is an example using vtkDecimatePro to decimate input polydata, if provided, or a sphere otherwise.
+    '''
+    parser = argparse.ArgumentParser(description=description, epilog=epilogue)
+    parser.add_argument('filename', nargs='?', default=None, help='Optional input filename e.g cow.g.')
+    parser.add_argument('reduction', nargs='?', type = float, default=None, help='Optional reduction percentage e.g 0.9')
+    args = parser.parse_args()
```

Comment:
Set the default to `0.9` and change the help text to: `Sets the decimation target reduction, (default is 0.9)`.